pss/pss_test.go

```diff
@@ -495,6 +495,133 @@ func TestAddressMatchProx(t *testing.T) {
 	}
 }
 
+func TestMessageEnqueued(t *testing.T) {
+
+	// setup
+	privkey, err := crypto.GenerateKey()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	addr := make([]byte, 32)
+	addr[0] = 0x01
+	ps := newTestPssStart(privkey, network.NewKademlia(addr, network.NewKadParams()), NewParams(), false)
+	outboxCapacity := 2
+	ps.outbox = make(chan *outboxMsg, outboxCapacity)
+
+	dequeue := func() *PssMsg {
+		select {
+		case oMsg := <-ps.outbox:
+			return oMsg.msg
+		default:
+			return nil
+		}
+	}
+
+	messageAddr := addr
+	topic := [4]byte{}
+	data := []byte{0x66, 0x6f, 0x6f}
+
+	msg := testMessage(messageAddr, topic, data)
```

Comment:
Maybe we can just have `testRandomMessage` and conceal the addr, topic, data within?

Reply:
Good idea. Fixed in commit 4e2aff6